kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/preprocess/FieldNameSanitizer.java

```diff
@@ -0,0 +1,31 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FieldNameSanitizer {
+
+    private static String sanitizeName(String name) {
+        name = name.replaceAll("^[^a-zA-Z_]+", "");
+        return name.replaceAll("[^a-zA-Z0-9_]", "_");
```

Comment:
we should pre-compile the regex patterns and store them as static final variables to speed things up.